docs/basics/part-2-outputs.md

```diff
@@ -0,0 +1,290 @@
+To copy the files for [this example ( Part 2 )](https://github.com/earthly/earthly/tree/main/examples/tutorial/go/part2) run
+
+```bash
+earthly --artifact github.com/earthly/earthly/examples/tutorial/go:main+part2/part2 ./part2
+```
+See below for examples in [Python](#more-examples), [Javascript](#more-examples) and [Java](#more-examples).
```

Comment:
I wonder if this part would look better in an info box, so that the reader can just skip straight to the meat if they want to.

Or maybe the text following simply needs a `##` title.

I was reading through this page as I realized that I wasn't sure where exactly it actually starts from.

Reply:
Good point. I was trying to avoid any gitbook specific markup for when we move off of it. I'll add a ##.